wordpress-mobile/WordPress-Android
# Set user-agent to Gutenberg WebView

This request was failing after login affecting Gutenberg Web View for Unsupported blocks.

Fixes parts of https://github.com/wordpress-mobile/gutenberg-mobile/issues/2358

Example of user-agent that we set to user-agent:
`user-agent: Mozilla/5.0 (Linux; Android 10; Pixel 3 Build/QQ2A.200501.001.B2; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/83.0.4103.106 Mobile Safari/537.36 wp-android/alpha-230`

To test:
- Have a post on your non-Atomic site ready with an unsupported block.
- Log out of WPAndroid app.
- Delete the app.
- Build from this PR.
- Run Metro server (to have the Unsupported blocks option available).
- Login and navigate to the post with the unsupported block.
- Open the post.
- Open the unsupported block on Gutenberg Web-View.
- Check that it doesn't prompt for login-in.


PR submission checklist:


cc @aerych
